src/test/resources/test-schema.sql

```diff
@@ -1,7 +1,7 @@
 drop table if exists proxy;
 
 CREATE TABLE proxy (
-    id integer NOT NULL AUTO_INCREMENT,
```

Comment:
The tests use postgresql, but this file was written for a different DB type. I updated the SQL to work with postgresql.